data/azure/instance_types.json

```diff
@@ -81,8 +102,10 @@
   "Standard_DS4_v2": {"up": "Standard_DS5_v2", "down": "Standard_DS3_v2", "superseded": {"regular": "Standard_D8s_v3"}},
   "Standard_DS5_v2": {"up": null, "down": "Standard_DS4_v2", "superseded": {"regular": "Standard_D16s_v3"}},
   "Standard_DS11_v2": {"up": null, "down": null, "superseded": {"regular": "Standard_E2s_v3"}},
-  "Standard_DS12_v2": {"up": "Standard_DS13_v2", "down": "Standard_DS11_v2", "superseded": {"regular": "Standard_E4s_v3"}},
-  "Standard_DS13_v2": {"up": "Standard_DS13-2_v2", "down": "Standard_DS12_v2", "superseded": {"regular": "Standard_E8s_v3"}},
+  "Standard_DS12_v2": {"up": "Standard_DS12-1_v2", "down": "Standard_DS11_v2", "superseded": {"regular": "Standard_E4s_v3"}},
+  "Standard_DS12-1_v2": {"up": "Standard_DS12-2_v2", "down": "Standard_DS12_v2", "superseded": {"regular": "Standard_DS12_v2"}},
+  "Standard_DS12-2_v2": {"up": "Standard_DS13_v2", "down": "Standard_DS12-1_v2", "superseded": {"regular": "Standard_DS12_v2"}},
```

Comment:
Can you cross-check the value for "superseded"

Reply:
Rolled back change